gaia-header.js

```diff
@@ -70,7 +70,7 @@ const MAXIMUM_FONT_SIZE = 23;
  * @return {Element} constructor
  */
 module.exports = component.register('gaia-header', {
-  rtl: true,
+  rtl: true, // triggers a workaround for bug 1100912 in gaia-component
```

Comment:
it is — I just reverted it to `rtl` in the second commit to see if that was the reason why the tests failed. I’ll remove this second commit when we land the patch on gaia-component.